### PR TITLE
Remove origin

### PR DESCRIPTION
**Pull request by thomasneirynck**
Remove the getOrigin method.

Revealed by https://github.com/elastic/kibana/pull/104422

Kibana used to merge configs from the kibana.yml and EMS into a single data-structure. With the removal of the legacy maps visualizations in Kibana, this is no longer the case.

The `ORIGIN` is an artefact from that approach. It is not required in the EMS-client.

### CODE REVIEW

File: src/ems_service.ts [hunk 1]
```diff
@@ -50,10 +48,6 @@ export abstract class AbstractEmsService implements IEmsService {
     return attributions.join('|');
   }
 
-  getOrigin(): string {
```
Comment (nickpeihl) [inline src/ems_service.ts:53]: `getOrigin` is still used in Kibana [here](https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/src/plugins/maps_ems/public/service_settings/service_settings.ts#L88) and [here](https://github.com/elastic/kibana/blob/6693ef371f887eca639b09c4c9b15701b4ebabd4/src/plugins/maps_ems/public/service_settings/service_settings.ts#L132). Are we expecting to remove usages there as well?

Comment (thomasneirynck) [reply src/ems_service.ts:53]: yes, we're removing this here https://github.com/elastic/kibana/pull/104422

Comment (nickpeihl) [reply src/ems_service.ts:53]: Ah, OK, I missed that.

**Review by nickpeihl — APPROVED** [review]
lgtm! code review

**Comment by jsanz** [comment]
@thomasneirynck this PR should be ready for merge, but not sure if this should also be backported to `8.0` or just stay in `master` for the next `8.1` release.

**Comment by thomasneirynck** [comment]
@jsanz maybe we should wait until https://github.com/elastic/kibana/pull/104422 merges. Otherwise, this would cause issues with upgrading Kibana to a newer version of the client.

apologies for the delay on that one..

**Comment by jsanz** [comment]
> Otherwise, this would cause issues with upgrading Kibana to a newer version of the client.

Since there are no automatic releases to `npm` for the `ems-client` I think it would be fine to merge this PR into `master` but it's also fine to wait, no problem!

**Comment by thomasneirynck** [comment]
https://github.com/elastic/kibana/pull/104422 merged, so we should be able to merge this one too.

**Comment by jsanz** [comment]
jenkins test this please

**Review by jsanz — APPROVED** [review]
@thomasneirynck 8.2 FF is getting close, since there are no blockers for this PR I think we can merge it right away.

Tested locally.

**Pull request merged**